packages/web/app/components/Translation/getTranslatedOptions.jsx

```diff
@@ -6,17 +6,18 @@ export const getTranslatedOptions = (options, prefix, TranslatedTextProps = {})
   if (!options) return [];
 
   return options.map((option, index) => {
-    const { label, ...rest } = option;
+    const { value, label, ...rest } = option;
     return typeof label === 'string'
       ? {
           label: (
             <TranslatedText
-              stringId={`${prefix}.${toCamelCase(label)}`}
+              stringId={`${prefix}.${toCamelCase(value)}`}
```

Comment:
This worked most of the time as often the `camelCase(label)` matched `camelCase(value)`, but when it didn't the translation wouldn't get picked up.